src/main/integrations/discord-presence/index.ts

```diff
@@ -50,23 +53,15 @@ function getSmallImageText(state: number) {
       return "Playing";
     }
 
-    case VideoState.Paused: {
-      return "Paused";
-    }
-
-    case VideoState.Buffering: {
-      return "Buffering";
-    }
-
     default: {
```

Comment:
This is providing information to the whole world about your buffering issue, returning buffering as paused seems more reasonable.

Reply:
I actually do agree that since this is just presence for the public to know this isn't necessary information to provide. I know why we added it initially but I'm open for this to change.